src/sentry/search/events/builder/__init__.py

```diff
@@ -1,5 +1,5 @@
+from .base import BaseQueryBuilder  # NOQA
```

Comment:
Why does it need NOQA? AFAIK we don't warn on relative imports, and this shouldn't warn of unused import, because it's in `__all__`

Reply:
🤷 just following the template, pretty sure this wasn't working before